sdk/servicebus/service-bus/test/stress/app/src/serviceBusStressTester.ts

```diff
@@ -160,7 +160,7 @@ export class ServiceBusStressTester {
         }
       } catch (error) {
         this.sendInfo.numberOfFailures++;
-        this.trackError("send", error);
+        this.trackError("send", error as Error);
```

Comment:
I prefer ` if (err instanceof Error) {` to narrow the type if `trackError()` is expecting an Error.